WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -10,7 +12,13 @@ public static class StartupHelper
 	{
 		private const string KeyPath = "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run";
 
-		public static void ModifyStartupSetting(bool runOnSystemStartup)
+		// Arguments to add Wasabi to macOS startup settings.
+		private static readonly string AddCmd = $"osascript -e \' tell application \"System Events\" to make new login item at end with properties {{name:\"{Constants.AppName}\", path:\"/Applications/{Constants.AppName}.app\", hidden:false}} \'";
```

Comment:
Just a question, why not declare these vars where they are used?

Reply:
Mostly because of readability, but I have the same question in my mind for a while. Not sure which one would be better.